schemas/github-pull-request-message.yml

```diff
@@ -0,0 +1,39 @@
+id: http://schemas.taskcluster.net/github/v1/github-pull-request-message.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "GitHub Pull Request Message"
+description: |
+  Message reporting that a GitHub pull request has occurred
+type:         object
+properties:
+  version:    {$const: message-version}
+  organizationName:
+    description: |
+      The GitHub `organizationName` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  repositoryName:
+    description: |
+      The GitHub `repositoryName` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
```

Comment:
You've previously defined this as `100`, for both organization and repository name